### ISSUE TITLE
Add tip to OneDrive documentation to avoid SharePoint throttling/blocking

### ISSUE THREAD

**Issue by olefrost** [rclone/rclone#5404]
#### The associated forum post URL from `https://forum.rclone.org`

https://forum.rclone.org/t/onedrive-errors/24677/13
https://forum.rclone.org/t/onedrvie-business-mount-problem/22561/6

#### What is your current rclone version (output from `rclone version`)?

v1.55.1

#### What problem are you are trying to solve?

Facilitate a great tip to users experiencing extensive throttling or blockage by SharePoint (OneDrive Business).

#### How do you think rclone should be changed to solve that?

Add a user agent tip from Microsoft to the [OneDrive Troubleshooting guide](https://rclone.org/onedrive/#troubleshooting) 

The wording could be something like:

> ##### Excessive throttling or blocked on SharePoint
> If you experience excessive throttling or is being blocked on SharePoint then it may help to set the user agent explicitly with a flag like this: `--user-agent "ISV|rclone.org|rclone/v1.55.1"`  
> 
> The specific details can be found in the Microsoft document: [Avoid getting throttled or blocked in SharePoint Online](https://docs.microsoft.com/en-us/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#how-to-decorate-your-http-traffic-to-avoid-throttling)
> 

**Thanks to @jcastill for finding and testing the tip.**

I can do the Pull Request if approved and don't mind if jcastill or somebody else wants to take the change from here. 

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by Cnly** [comment]
Thanks @olefrost, that sounds like a nice-to-have improvement to me! :) And it'd be great if you're also planning to do the PR.
